journals/journal_abbreviations_acs.csv

```diff
@@ -1,1739 +1,1739 @@
-Trends in Neurosciences;Trends Neurosci.
-Trends in Pharmacological Sciences;Trends Pharmacol. Sci.
-Trends in Plant Science;Trends Plant Sci.
-Tribology Transactions;Tribol. Trans.
-Turkish Journal of Chemistry;Turk. J. Chem.
-Ukrainian Biochemical Journal;Ukr. Biochem. J.
-Ukrainica Bioorganica Acta;Ukr. Bioorg. Acta
-Ukrainskii Khimicheskii Zhurnal (Russian Edition);Ukr. Khim. Zh. (Russ. Ed.)
-Ultramicroscopy;Ultramicroscopy
-Ultrasonics Sonochemistry;Ultrason. Sonochem.
-Vacuum;Vacuum
-Vascular Pharmacology;Vasc. Pharmacol.
-Vestnik Moskovskogo Universiteta, Seriya 2: Khimiya;Vestn. Mosk. Univ., Ser. 2: Khim.
-Vestnik Moskovskogo Universiteta, Seriya 2: Khimiya;Vestn. Mosk. Univ., Ser. 2: Khim.
-Vestsi Natsyyanal'nai Akademii Navuk Belarusi, Seryya Khimichnykh Navuk;Vestsi Nats. Akad. Navuk Belarusi, Ser. Khim. Navuk
-Vibrational Spectroscopy;Vib. Spectrosc.
-Viral Immunology;Viral Immunol.
-Virology;Virology
-Virus Genes;Virus Genes
-Virus Research;Virus Res.
-Water Environment Research;Water Environ. Res.
-Water Quality Research Journal of Canada;Water Qual. Res. J. Can.
-Water Research;Water Res.
-Water Resources;Water Resour.
-Water, Air, & Soil Pollution;Water, Air, Soil Pollut.
-Wear;Wear
-Women's Health;Women's Health
-Wood Science and Technology;Wood Sci. Technol.
-Wood and Fiber Science;Wood Fiber Sci.
-World Journal of Condensed Matter Physics;World J. Condens. Matter Phys.
-World journal of virology; World J. Virol.
-Wuji Huaxue Xuebao;Wuji Huaxue Xuebao
-Wuli Huaxue Xuebao;Wuli Huaxue Xuebao
-Wuli Xuebao;Wuli Xuebao
-X-Ray Spectrometry;X-Ray Spectrom.
-X-Ray Structure Analysis Online;X-Ray Struct. Anal. Online
-Xenobiotica;Xenobiotica
-Yakhak Hoechi;Yakhak Hoechi
-Yakugaku Zasshi;Yakugaku Zasshi
-Yeast;Yeast
-Yingxiang Kexue Yu Guang Huaxue;Yingxiang Kexue Yu Guang Huaxue
-Yingyong Huaxue;Yingyong Huaxue
-Yosetsu Gakkai Ronbunshu;Yosetsu Gakkai Ronbunshu
-Youji Huaxue;Youji Huaxue
-Yuki Gosei Kagaku Kyokaishi;Yuki Gosei Kagaku Kyokaishi
-Zeitschrift fuer Anorganische und Allgemeine Chemie;Z. Anorg. Allg. Chem.
-Zeitschrift fuer Kristallographie - Crystalline Materials;Z. Kristallogr. - Cryst. Mater.
-Zeitschrift fuer Kristallographie - New Crystal Structures;Z. Kristallogr. - New Cryst. Struct.
-Zeitschrift fuer Lebensmittel-Untersuchung und -Forschung A: European Food Research and Technology;Z. Lebensm.-Unters. Forsch. A
-Zeitschrift fuer Naturforschung, A: Physical Sciences;Z. Naturforsch., A: Phys. Sci.
-Zeitschrift fuer Naturforschung, B: A Journal of Chemical Sciences;Z. Naturforsch., B: J. Chem. Sci.
-Zeitschrift fuer Naturforschung, C: Journal of Biosciences;Z. Naturforsch., C: J. Biosci.
-Zeitschrift fuer Physikalische Chemie (Muenchen, Germany);Z. Phys. Chem. (Muenchen, Ger.)
-Zhipu Xuebao;Zhipu Xuebao
-Zhongguo Shengwu Huaxue Yu Fenzi Shengwu Xuebao;Zhongguo Shengwu Huaxue Yu Fenzi Shengwu Xuebao
-Zhongguo Wuji Fenxi Huaxue;Zhongguo Wuji Fenxi Huaxue
+"ACS Applied Materials & Interfaces","ACS Appl. Mater. Interfaces"
```

Comment:
Do we really need `"` everywhere? Wouldn't it be possible to use enclsoing quotes only when needed?

Reply:
Honestly, it's my preference, I'm used to making all the formatting consistent, and when there's a value that needs to have quotes added to it, I always want to make the other values consistent with that value. Not only in CSV documents, but also when I format js objects.
I don't think all the quotes affect the readability of the data, it clearly tells the user that what is between each pair of quotes is continuous text.
If the JabRef team thinks that "use quotes only when necessary" is a better solution, I can change it over.

Reply:
We discussed internally and follow your flow!